src/context/WalletProvider/KeepKey/components/WipedSuccessfully.tsx

```diff
@@ -43,7 +43,7 @@ export const WipedSuccessfully = () => {
           isFullWidth
           size='lg'
           onClick={handleRecoverWalletPress}
-          disabled={loading && true} // Un-disable when recover wallet is ready
+          disabled={loading || true} // Un-disable when recover wallet is ready
```

Comment:
WTF? this will always be true.

Reply:
I guess the comment is the `@TODO`, actually it should always be disabled until the recover flow will be implemented in the application

Reply:
Yeah, it should really be a TODO: https://github.com/shapeshift/web/pull/1490/commits/bfcd85aa6387b109d8da0b3e2adea927dc39462c